server/tests/Mailer.test.ts

```diff
@@ -76,19 +76,19 @@ describe('MailerService Class', () => {
     );
   });
 
-  it('Should provide a blank string if backup text is undefined', () => {
-    const mailer = new MailerService(data);
-    expect(mailer.backupText).toEqual('');
-  });
+  // it('Should provide a blank string if backup text is undefined', () => {
+  //   const mailer = new MailerService();
+  //   expect(mailer.backupText).toEqual('');
+  // });
 
-  it('Should default iCalEvent to undefined', () => {
-    const mailer = new MailerService(data);
-    expect(mailer.iCalEvent).toBeUndefined();
-  });
+  // it('Should default iCalEvent to undefined', () => {
+  //   const mailer = mailerService.loadValues(data);
+  //   expect(mailer.iCalEvent).toBeUndefined();
+  // });
 
-  it('Should log a warning if emailUsername, emailPassword, or emailService is not specified', () => {
-    // stub(Utilities, 'allValuesAreDefined').callsFake(() => false);
-    new MailerService(data);
-    expect(console.warn).toHaveBeenCalledTimes(1);
-  });
+  // it('Should log a warning if emailUsername, emailPassword, or emailService is not specified', () => {
+  //   // stub(Utilities, 'allValuesAreDefined').callsFake(() => false);
+  //   const mailer = new MailerService();
+  //   expect(console.warn).toHaveBeenCalledTimes(1);
+  // });
```

Comment:
```suggestion
  it('Should log a warning if emailUsername, emailPassword, or emailService is not specified', () => {
    new MailerService();
    expect(console.warn).toHaveBeenCalledTimes(1);
  });
```
While the validation can definitely be improved, I think this is better than nothing.